src/com/lftechnology/java_training/dec31/DateTest.java

```diff
@@ -0,0 +1,148 @@
+package com.lftechnology.java_training.dec31;
+
+import java.util.Scanner;
+
+/**
+ * Write a test application named DateTest that demonstrates class Date's
+ * capabilities.
+ * 
+ * @author utsabn34
+ * 
+ */
+public class DateTest {
+
+	public static void main(String[] args) {
+		Date date1 = new Date();
+		Date date2 = new Date();
+		date1.assignValue();
+		date2.setDay(12);
+		date2.setMonth(01);
+		date2.setYear(2013);
+		System.out.println("User inputted  is: " + date1.getDay() + "/"
+				+ date1.getMonth() + "/" + date1.getYear());
+		date2.displayDate();
+	}
+
+}
+
+/**
+ * a class called Date that includes three pieces of information as instance
+ * variables a month (type int), a day (type int) and a year (type int). Your
+ * class should have a constructor that initializes the three instance variables
+ * and assumes that the values provided are correct. Provide a set and a get
+ * method for each instance variable. Provide a method displayDate that displays
+ * the month, day and year separated by forward slashes (/).
+ * 
+ * @author utsabn34
+ * 
+ */
+class Date {
```

Comment:
Ok that is fine for time being but better to use name like DateUtility because we have already class name Date of java utils and java sqls , this might get confused, from next time keep that in mind :)